pkgs/by-name/th/thc-secure-delete/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, installShellFiles
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "thc-secure-delete";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gordrs";
+    repo = "thc-secure-delete";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-hvWuxFkzhOSCplPtyjRtn36bIk6KdPBcpr3lAmiAyfE=";
+  };
+
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+
+  makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install -d $out/bin
+    install -m755 srm sdmem sswap sfill $out/bin/
```

Comment:
```suggestion
    install -Dm755 srm sdmem sswap sfill -t $out/bin/
```

Reply:
I previously tried this method but package build fails so i use the install -d and install -m saperately,

Reply:
note the `-D` and `-t`.
it builds fine on my end.

Reply:
Yeah, the build works with -t.